src/testing/common-errors.md

```diff
@@ -136,19 +166,20 @@ RenderViewport#5a477 NEEDS-LAYOUT NEEDS-PAINT NEEDS-COMPOSITING-BITS-UPDATE
 
 **How might you run into this error?**
 
-Usually, the issue is related to violation of box constraints, and it needs to
-be solved by providing more information to Flutter about how you’d like to
-constrain the widgets in question. You can learn more about how constraints work
-in Flutter on the page [Understanding constraints][]. 
-
-[Understanding constraints]: {{site.url}}/ui/layout/constraints
+Usually, the issue is related to violation of box constraints,
+and it needs to be solved by providing more information
+to Flutter about how you’d like to constrain the widgets in question.
+You can learn more about how constraints work
+in Flutter on the [Understanding constraints][] page. 
 
-The `RenderBox was not laid out` error is often caused by one of two other errors:
+The `RenderBox was not laid out` error is often
+caused by one of two other errors:
 
 * ‘Vertical viewport was given unbounded height’
 * ‘An InputDecorator...cannot have an unbounded width’
 
-## ‘Vertical viewport was given unbounded height’
+
+## <a href="unbounded">‘Vertical viewport was given unbounded height’</a>
```

Comment:
To simplify this and fix the error, just add a separate anchor before the header:

```suggestion
<a id="unbounded"></a>
## ‘Vertical viewport was given unbounded height’
```